packages/stan/src/registry.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * External dependencies
+ */
+import { noop, isObject } from 'lodash';
+
+/**
+ * @typedef {( atomState: import('./types').WPAtomState<any> ) => void} RegistryListener
+ */
+
+/**
+ * @template T
+ * @param {import('./types').WPAtom<any>|import('./types').WPAtomFamilyItem<T>} maybeAtomFamilyItem
+ * @return {boolean} maybeAtomFamilyItem is WPAtomFamilyItem<T>.
+ */
+export function isAtomFamilyItem( maybeAtomFamilyItem ) {
+	if (
```

Comment:
Could it `return` immediately instead?
```suggestion
	return (
```

Not that important though.